<launch>
	<arg name="veh" default="lily"/>
	<arg name="config" default="baseline"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<arg name="local" default="true" doc="true to launch locally on laptop. false to launch of vehicle"/>
	<arg name="pkg_name" default="mocap" doc="name of the package"/>
	<arg name="node_name" default="mocap_waypoint_planning_node" doc="name of the node"/>
	<arg name="get_pose_node" default="get_model_pose" doc="name of the node"/>

	<group ns="$(arg veh)">

		<!-- remap from="mocap_waypoint_planning_node/vehicle_pose_pair" to="mocap_localization_node/vehicle_pose_pair"/ -->
	    <!-- Local -->
	    <node if="$(arg local)" pkg="$(arg pkg_name)" type="$(arg get_pose_node).py" name="$(arg get_pose_node)" output="screen" clear_params="true" required="true">
	    	
	    </node>
	        <!---rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/-->
	  

	    <node if="$(arg local)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	    </node>

	    <!-- Remote -->
		<include unless="$(arg local)" file="$(find duckietown)/machines"/>
	    <node unless="$(arg local)" machine="$(arg veh)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true">
	        <!--rosparam command="load" file="$(find duckietown)/config/$(arg config)/$(arg pkg_name)/$(arg node_name)/$(arg param_file_name).yaml"/-->
	    </node>
	</group>

</launch>